backend/main_test.go

```diff
@@ -80,7 +80,7 @@ func TestPostPeriod(t *testing.T) {
 
 	teamID := "myteam"
 	addTeam(handler, teamID, t)
-	periodJSON := `{"id":"2019q1","displayName":"2019Q1","unit":"person weeks","notesURL":"http://test","buckets":[{"displayName":"Bucket one","allocationPercentage":80,"objectives":[{"name":"Objective 1","resourceEstimate":0,"commitmentType":"Committed","notes":"some notes","assignments":[]}]}],"people":[]}`
+	periodJSON := `{"id":"2019q1","displayName":"2019Q1","unit":"person weeks","notesURL":"http://test","buckets":[{"displayName":"Bucket one","allocationPercentage":80,"objectives":[{"name":"Objective 1","resourceEstimate":0,"commitmentType":"Committed","notes":"some notes","assignments":[]}]}],"people":[{"ID": "alice", "DisplayName": "Alice Atkins", "Location": "LON", "Availability": 5}]}`
```

Comment:
I was actually quite surprised this worked, since the keys you've used for the person in your JSON don't match the tags in the `struct` (the case is different, e.g. `ID` versus `id`).

Reading [the Go json docs](https://golang.org/pkg/encoding/json/#Unmarshal), this is apparently expected, since the unmarshalling prefers an exact match but will fall back to a case-insensitive match if one is available.

However, it would be better to use JSON which matches the kind the UI will produce. So please use JSON keys that exactly match the `json:` tags in the structs.